scripts/bake-scalable-azure-image-pipeline.ps1

```diff
@@ -93,6 +93,6 @@ Bake-IdeMsi -VersionText $VersionText `
             -Pipeline:$true `
             -KeyPairName $KeyPairName `
             -KeyPairPath $KeyPairPath `
-            -GitUserName $GitUserName `
+            -GitUserName 'praveen-acc' `
```

Comment:
This change in the source code is not OK. Anybody using this is now locked into `praveen-acc`. Why are you not changing the pipeline to pass in your GitHub user name? I know its required to make temporary changes like this because its your repo that has the changes you are making. But the whole point of introducing this parameter was so that each developer can provide THEIR OWN user name.

So, change the pipeline and revert this commit.